tests/unit/test_baseclient.py

```diff
@@ -199,7 +199,7 @@ def test_client_certs_mocked(self, mock_listdir):
             [".DS_Store", "client-2048.crt", "client-2048.key"]
         )
```

Comment:
It would be good to add another test that confirms that `certs=“/tmp”` is correct